nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/RuntimeDeployment.java

```diff
@@ -115,6 +118,26 @@ protected void index(LocalDeploy config) {
         }
     }
 
+    /**
+     * @since 9.1
+     * @param config
+     */
+    protected void index(PartialDeploy config) {
+        if (config == null) {
+            return;
+        }
+
+        partialBundles.putIfAbsent(config.bundle(), new HashSet<>());
+        Set<TargetExtensions> pairs = partialBundles.get(config.bundle());
```

Comment:
You can directly write this: `partialBundles.computeIfAbsent(config.bundle(), key ->new HashSet<>())` which avoid to instantiate a HashSet if value exists.